src/apps/utep_sdg_11_3_1/components/App/Map/index.js

```diff
@@ -0,0 +1,157 @@
+import { connect } from 'react-redux';
+import Select from '../../../state/Select';
+import Action from "../../../../../state/Action";
+
+import {cloneDeep, isEqual} from 'lodash';
+
+import wrapper from '../../../../../components/common/maps/MapWrapper';
+
+import utils from '../../../../../utils/utils';
+import { quartilePercentiles, mergeAttributeStatistics } from '../../../../../utils/statistics';
+
+const useActiveMetadataKeys = {
+	scope: false,
+	attribute: false,
+	period: true
+};
+
+const mapStateToProps = (state, props) => {
+	let namesFilter = {};
+	let filter = {};
+	let chartCfg = {};
+
+	return (state) => {
+		let layersState = Select.maps.getLayersStateByMapKey(state, props.mapKey, useActiveMetadataKeys);
+		let layersData = layersState ? layersState.map(layer => {
+			const filter = cloneDeep(layer.mergedFilter)
+			return {filter, data: layer.layer}
+		}) : null;
+		let layers = Select.maps.getLayers(state, layersData);
+		// let vectorLayers = layers ? layers.filter((layerData) => layerData.type === 'vector') : [];
```

Comment:
Ty zakomentované věci tam být asi nemusí

Reply:
Ještě tam jdu dodělat vektoroubou vrstvu, pak to pročistím

Reply:
Ok